packages/ilib-lint/src/plugins/utils.js

```diff
@@ -23,25 +23,28 @@ import { xml2js } from 'xml-js';
 * Extracts the XLIFF version from the provided data.
 *
 * @param {String} data The XML data as a string.
-* @returns {Object} the xliff version and style.
+* @returns {Object} the xliff version, style and sourceLocale.
 */
 export function getXliffInfo(data) {
     const defaultInfo = {
         version: "1.2",
-        style: "standard"
+        style: "standard",
+        sourceLocale: "en-US"
     };
     if (!data) return defaultInfo;
 
     try {
         const parsedData = xml2js(data);
         const xmlVersion = parsedData?.elements?.[0]?.attributes?.version;
         const projectAttr = parsedData?.elements?.[0]?.elements?.[0]?.attributes?.['l:project'];
+        const sourceLocale = parsedData?.elements?.[0]?.attributes?.srcLang;
```

Comment:
Does this work with xliff 1.2 ? I think 1.2 uses "sourceLanguage" instead of "srcLang"

Reply:
I updated codes and added test file as well. Please check them again.